source/core/coverage/ut_coverage_helper.pkb

```diff
@@ -59,17 +68,18 @@ create or replace package body ut_coverage_helper is
   end;
 
   procedure coverage_stop is
-    l_return_code binary_integer;
   begin
     if not g_develop_mode then
```

Comment:
Is it correct to check for develop here? We are stoping coverage regardless of the mode, aren't we?